src/components/VApp/VApp.js

```diff
@@ -32,6 +32,9 @@ export default {
   },
 
   computed: {
+    application () {
+      return this.$vuetify.application
+    },
```

Comment:
Should this be removed? I added it but I'm not sure.

Reply:
I don't see any reason to leave it in.